gen/PathPlannerTrajectory.yml

```
@@ -5,31 +5,69 @@ classes:
     methods:
       PathPlannerTrajectory:
         overloads:
-          std::vector<Waypoint>, units::meters_per_second_t, units::meters_per_second_squared_t, bool:
-          std::vector<PathPlannerState>:
+          std::vector<Waypoint>&, std::vector<EventMarker>&, PathConstraints, bool:
```

`Waypoint` appears to be private now. This constructor overload is `protected` in the Java version.
```suggestion
          std::vector<Waypoint>&, std::vector<EventMarker>&, PathConstraints, bool:
          	ignore: true  # private Waypoint class, Java protected overload
```